doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -358,6 +358,10 @@ Wi-Fi samples
 Other samples
 -------------
 
+* :ref:`multicore_hello_world` sample:
+
+  * Added :ref:`zephyr:sysbuild` support to the sample.
+
```

Comment:
Since the location of multicore_hello_world sample is now changed, the same needs to be changed in the change log. We could have a new section heading "Multicore samples" like we have for "Other samples" and move the whole entry to that section.

Reply:
I will move it just before nrf5340 samples